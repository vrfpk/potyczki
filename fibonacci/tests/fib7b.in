126929481310879
