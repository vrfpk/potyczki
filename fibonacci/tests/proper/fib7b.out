2561267575684558
